src/bin/vip-config.js

```diff
@@ -2,11 +2,28 @@
 
 import command from '../lib/cli/command';
 
+const usage = 'vip config';
+const exampleUsage = 'vip @example-app.develop config';
+
+// Command examples
+const examples = [
+	{
+		usage: `${ exampleUsage } software get`,
+		description: 'Retrieve the current versions of all environment software.',
+	},
+	{
+		usage: `${ exampleUsage } envvar list`,
+		description: 'List the names of all environment variables on the environment.',
+	},
+];
```

Comment:
Another small nit, but the examples are in the opposite order of the commands for this one.